Dockerfile

```diff
@@ -0,0 +1,22 @@
+# Use an official Node.js runtime as a parent image
+FROM node:14
+
+# Set the working directory to /app
+WORKDIR /app
+
+# Copy package.json and package-lock.json to the container
+COPY package*.json ./
+
+# Install project dependencies
+RUN npm install
+
+# Copy the rest of the application code to the container
+COPY . .
+
+# Expose the port that your application listens on
+EXPOSE 8000
+
+EXPOSE 8080
+
+# Define the command to run your application
+CMD ["npm", "start"]
```

Comment:
Should I use yarn?

Reply:
> Should I use yarn?
This project does not use yarn at the moment. 

1. There is an [existing `ENTRYPOINT` script in the node image which you should use directly](https://github.com/nodejs/docker-node/blob/main/docker-entrypoint.sh). 
2. npm should never be used to start a Node application because [npm swallows the exit signals that container runtimes send](https://github.com/nodejs/docker-node/blob/main/docs/BestPractices.md#cmd), which can and will interfere with those runtimes trying to stop & gracefully shutdown containers running git-proxy.

```suggestion
CMD ["index.js"]
```